src/utils.js

```diff
@@ -32,24 +31,59 @@ export const findAvailablePort = start =>
 // Retrieves the static.config.js from the current project directory
 export const getConfig = () => {
   const config = require(path.resolve(path.join(process.cwd(), 'static.config.js'))).default
+
+  // path defaults
+  config.paths = {
+    src: 'src',
+    dist: 'dist',
+    public: 'public',
+    ...(config.paths || {}),
+  }
+
+  // Resolve the root of the project
+  const ROOT = path.resolve(process.cwd())
+
+  // Use the root to resolve all other relative paths
+  const resolvePath = relativePath => path.resolve(path.join(ROOT, relativePath))
+
+  // Resolve all paths
+  const paths = {
+    ROOT,
+    LOCAL_NODE_MODULES: path.resolve(__dirname, '../node_modules'),
+    SRC: resolvePath(config.paths.src),
+    DIST: resolvePath(config.paths.dist),
+    PUBLIC: resolvePath(config.paths.public),
+    NODE_MODULES: resolvePath('node_modules'),
+    PACKAGE: resolvePath('package.json'),
+    HTML_TEMPLATE: path.join(resolvePath('dist'), 'index.html'),
```

Comment:
Really cool stuff!
Maybe this should be `HTML_TEMPLATE: path.join(resolvePath(config.paths.dist), 'index.html')`, or am I misunderstanding something?

Reply:
You're right, it should :(